Classes/Domain/Model/Book.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Books\Domain\Model;
+
+final class Book
+{
+    /** @var string */
+    private $title;
+
+    /** @var int */
+    private $yearOfPublication;
```

Comment:
These two need to have default values in the correct type so that new models still have working getters.